backend/schema/validate.go

```diff
@@ -613,20 +643,72 @@ func validateVerbMetadata(scopes Scopes, module *Module, n *Verb) (merr []error)
 	return
 }
 
-func validateIngressRequestOrResponse(scopes Scopes, module *Module, n *Verb, reqOrResp string, r Type) (fieldType Type, body Symbol, merr []error) {
+type HTTPRequestExtractedTypes struct {
```

Comment:
This should be private
```suggestion
type httpRequestExtractedTypes struct {
```